knowledge-base/map-center-on-selected-marker.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Center the Map on a Selected Marker
+description: Learn how to center a Telerik Map for Blazor component on a marker selected from a list.
+type: how-to
+page_title: How to Center a Map on Marker in Blazor Applications
+slug: map-center-on-selected-marker
+tags: map, blazor, center, marker, selection, list
+res_type: kb
+ticketid: 1671734
+---
+
+## Environment
+
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Map for Blazor</td>
+	    </tr>
+    </tbody>
+</table>
+
+## Description
+
+There's a list of markers. I need to center the map on a marker when it's selected from the list. 
+
+- How can I dynamically update the center of a map based on a marker's coordinates?
+
+## Solution
+
+To center the map on a specific marker's coordinates, use a variable to bind the `Center` parameter of the `TelerikMap` component. Update this variable whenever a selection is made from the list. Here's how you can implement this:
+
+1. Use a component like the `TelerikDropDownList` to display the list of markers. Handle the `OnChange` event to update the map's center.
+
+2. In the `OnChange` event handler, update the map's `Center` property to the coordinates of the selected marker.
+
+````RAZOR
```

Comment:
Use

* `caption`
* capital letters for component properties
* `nameof()` for field names
* two-way binding for Map `Zoom`

Reply:
All comments are addressed except for the two-way binding for Zoom as the Map does not support it.